WordPress/Classes/Networking/BlogServiceRemote.h

```diff
@@ -4,9 +4,14 @@
 
 typedef void (^OptionsHandler)(NSDictionary *options);
 typedef void (^PostFormatsHandler)(NSDictionary *postFormats);
+typedef void (^MultiAuthorCheckHandler)(BOOL isMultiAuthor);
 
 @protocol BlogServiceRemote <NSObject>
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog
+                        success:(MultiAuthorCheckHandler)success
+                        failure:(void (^)(NSError *error))failure;
```

Comment:
> If the user is not an administrator, this will probably fail both in REST and XML-RPC

Good catch. For XML-RPC I missed that wp.getUsers required the `list_users` capability. `wp.getAuthors` is probably the one I want instead.  For the REST API, a quick test shows that users with an Administrator or an Editor role will be able to retrieve a list of authors. The change to the XML-RPC will match that behavior, which is what we want. 

> have you considered alternatives to the extra network call

I think we need to be batching calls were we can to reduce the total number.  We have some good candidates for this but I think those changes belong it its own PR.